.github/workflows/ci-treesitter.yml

```diff
@@ -36,6 +36,8 @@ jobs:
       - name: Make tree-sitter-essence the root directory
         run: cp -r repo/crates/tree-sitter-essence/* .
 
+      # not sure if this needs rust-cache?
+
```

Comment:
Wouldn't hurt :)

Reply:
If it is any hassle to add, I say lets not do it - the benefits of caching here would be quite small if any.

Reply:
I presume caches with identical keys are overwritten on change not combined, which is why scopes would be needed?

Reply:
that's my worry. when this smaller job finishes it could remove some useful stuff from the cache. however, if it starts with a non-empty cache and adds onto it, perhaps it won't remove anything. hard to know without either learning a bit more about the action or experimenting with it.